cli/cmd/common.go

```diff
@@ -22,3 +23,13 @@ func handleCmdErr(cmd *cobra.Command, err error) {
 		log.Fatalf(err.Error())
 	}
 }
+
+var errNoConfig = errors.New("tt.yaml not found, you need to create tt" +
+	" environment config with tt init")
+
+func checkConfig(cmdCtx *cmdcontext.CmdCtx) error {
```

Comment:
i would like to see small description to function

Reply:
Added.